scripts-available/CDB_Quota.sql

```diff
@@ -1,3 +1,22 @@
+CREATE OR REPLACE FUNCTION cartodb._CDB_total_relation_size(_schema_name TEXT, _table_name TEXT)
+RETURNS bigint AS
+$$
+BEGIN
+  IF EXISTS (
+      SELECT 1 FROM information_schema.tables
+      WHERE table_catalog = current_database()
+        AND table_schema = _schema_name
+        AND table_name = _table_name
+  )
+  THEN
+    RETURN pg_total_relation_size(format('"%s"."%s"', _schema_name, _table_name));
```

Comment:
I'm not fully sure. As an alternative I had:

```sql
CREATE OR REPLACE FUNCTION cartodb._CDB_total_relation_size(_schema_name TEXT, _table_name TEXT)
RETURNS bigint AS
$$
DECLARE relation_size bigint;
BEGIN
  relation_size := 0;
  SELECT pg_total_relation_size(format('"%s"."%s"', _schema_name, _table_name)) INTO relation_size WHERE EXISTS (
      SELECT 1 FROM information_schema.tables
      WHERE table_catalog = current_database()
        AND table_schema = _schema_name
        AND table_name = _table_name
  );
  return relation_size;
END;
$$
LANGUAGE 'plpgsql' VOLATILE;
```

But it's kind of the same. The alternative is use a EXCEPTION catch but I was trying to avoid it.

cc @pramsey any thoughts?